src/services/db/review.ts

```diff
@@ -83,3 +90,45 @@ export const approvePullRequest = (
       },
     }
   )
+
+export const getComments = async (
+  siteName: string,
+  pullRequestNumber: number
+) => {
+  const rawComments = await axiosInstance
+    .get<RawComment[]>(`${siteName}/issues/${pullRequestNumber}/comments`)
+    .then(({ data }) => data)
+  return _.compact(
+    rawComments.map((rawComment) => {
+      try {
+        const commentData = JSON.parse(rawComment.body)
+        const { user, message } = commentData
+        return {
+          user,
+          message,
+          createdAt: rawComment.created_at,
+        }
+      } catch (e) {
+        // Not properly formatted comment, ignore
+        return null
```

Comment:
i checked in `ReviewRequestService` and this doesn't seem to be reflected, which means that this is a dangerous return. could we annotate the return type on the function explicitly

Reply:
The nulls are actually removed by `_.compact`! I've added comments to make it clearer